components/places/src/storage/bookmarks/root_guid.rs

```diff
@@ -28,23 +30,23 @@ lazy_static! {
     static ref GUIDS: [(BookmarkRootGuid, SyncGuid); 5] = [
         (
             BookmarkRootGuid::Root,
-            SyncGuid(BookmarkRootGuid::Root.as_str().into())
+            SyncGuid::from_str(BookmarkRootGuid::Root.as_str()).unwrap()
```

Comment:
This is gross, but it was gross before  and we can address in a follow up.

That said, probably use `SyncGuid::from(...)` instead.